app/assets/stylesheets/application.scss

```diff
@@ -64,7 +64,7 @@ h3.quote:before, h3.quote:after {
 
 input {
   font-size: 18px;
-  padding: 10px 10px 10px 5px;
+  padding: 10px 10px 10px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px` instead of `10px 10px 10px 10px`